src/EPR.Payment.Service/Services/RegistrationFees/Producer/ProducerFeesCalculatorService.cs

```diff
@@ -36,11 +39,12 @@ public async Task<RegistrationFeesResponseDto> CalculateFeesAsync(ProducerRegist
                 {
                     ProducerRegistrationFee = await _baseFeeCalculationStrategy.CalculateFeeAsync(request, cancellationToken),
                     ProducerOnlineMarketPlaceFee = await _onlineMarketCalculationStrategy.CalculateFeeAsync(request, cancellationToken),
+                    ProducerLateRegistrationFee = await _lateFeeCalculationStrategy.CalculateFeeAsync(request, cancellationToken),
                     SubsidiariesFeeBreakdown = await _subsidiariesFeeCalculationStrategy.CalculateFeeAsync(request, cancellationToken)
                 };
 
                 response.SubsidiariesFee = response.SubsidiariesFeeBreakdown.TotalSubsidiariesOMPFees + response.SubsidiariesFeeBreakdown.FeeBreakdowns.Select(i => i.TotalPrice).Sum();
-                response.TotalFee = response.ProducerRegistrationFee + response.ProducerOnlineMarketPlaceFee + response.SubsidiariesFee;
+                response.TotalFee = response.ProducerRegistrationFee + response.ProducerOnlineMarketPlaceFee + response.SubsidiariesFee + response.ProducerLateRegistrationFee ;
                 response.PreviousPayment = 0;// TODO: This will not be 0 but calculated once the database schema is changes are ready
```

Comment:
Very small change, but please remove space between ProducerLateRegistrationFee and ; ie
response.ProducerLateRegistrationFee;